assignment/index.html

```diff
@@ -62,10 +69,61 @@
 
       ===================== */
 
-      var jsonToCsv = function(json) { console.log(json); };
-
-
-      var addMarkers = function(map) {};
+      var jsonToCsv = function(json) {
+        let hc = [];
+
+        let keys = [];
+        for (let i = 0; i < json.length; i++) {
+          keys = keys.concat(Object.keys(json[i]));
```

Comment:
Will this be generating a new value for `keys` every time it loops through for each element in `json`?

I may be misreading it. But seems like that would only need to happen once.

Reply:
It appends all keys from each element to `keys`, then removes duplicates-- I was trying to account for the possibility that not every element had the same keys.

It doesn't really matter though since each element of `json` has the same keys, and some later parts of my code can't handle inconsistent keys either.